Cassity_Zachary/Assignments/Registration Form/rf.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, render_template, request, redirect,session,flash
+import re
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+NAME_REGEX = re.compile(r'^[a-zA-Z]+$')
+app = Flask(__name__)
+app.secret_key = "ThisIsSecret!"
+valid = True
+@app.route('/')
+def main_page():
+    return render_template('rf.html')
+
+@app.route('/submit',methods=['POST'])
+def submit():
+    if (len(request.form['email'] and request.form['first_name'] and request.form['last_name'] and request.form['password']) < 1):
```

Comment:
Best to seperate out the fields even though you're checking for the same condition.  Also, with them all being combined with 'and' what if only one of the four fields was blank?